style.scss

```diff
@@ -5,4 +5,5 @@
 }
 body {
   color: purple;
+  color: white;

```

Comment:
zasto imamo dva color propertija? jedan je visak, ispravi to molim te

Reply:
Ispravljena greska :)